### ISSUE TITLE
Httpget error

### ISSUE THREAD

**Issue by sunwangshu** [genekogan/p5js-osc#8]
I was trying to follow the "osc in p5.js" instructions on a Windows machine because I wanted to use Kinect. Because it was borrowed from ER I didn't have any screenshot yet.

I think I was using Node 6.x or something. 
Then I tried gitclone the folder. It looks good. 
Then I used powershell to do npm install, then "node bridge.js".

I compared the demo video and **noticed that I didn't get the "socket.io started" message in the video, and didn't get any error message either**. The cursor only stayed leftmost in a new empty line below "node bridge.js" until I do Ctrl +C to stop it, then I can input again.

Then I run the Processing sketch in example p5-basic, and then double clicked index.html and opened it in Chrome. Even if I move the circle in Processing, the p5 sketch only shows a blue square appeared without the circle. And I opened the inspect, only finding a lot of httpget errors.

**Comment by genekogan** [comment]
yes, i am getting this behavior too. possibly something changed in socket that i need to update. i won't be able to look into this for at least a week or two, but will revisit this issue and try to resolve it. 

if anyone else has seen this issue come up and knows how to resolve it, please share here.

**Comment by iverson777** [comment]
I got similar error and I just modify sketch.js 
`var socket = io.connect('http://127.0.0.1:'  ......`
to 
`var socket = io.connect('http://127.0.0.1:8081'  ......`

**Comment by genekogan** [comment]
@iverson777 thanks! this appears to fix it. i'm updating all the examples. cc @sunwangshu
